packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -694,6 +697,53 @@ class _FloatingActionButtonTransitionState extends State<_FloatingActionButtonTr
 /// ```
 /// {@end-tool}
 ///
+/// ## Scaffold layout, the keyboard, and display "notches"
+///
+/// The scaffold will expand to fill the available space. That usually
+/// means that it will occupy its entire window or device screen. When
+/// the device's keyboard appears the Scaffold's ancestor [MediaQuery]
+/// widget's [MediaQueryData.viewInsets] changes and the Scaffold will
+/// be rebuilt. By default the scaffold's [body] is resized to make
+/// room for the keyboard. To prevent the resize set
```

Comment:
I would think
`room for the keyboard. To prevent the resize, set`
but my english is crappy so you decide

Reply:
I'm not sure about the "right" comma usage here. If you think the comma makes it easier to understand, I'm happy to add one.